packages/node_modules/@webex/react-component-utils/src/constants.js

```diff
@@ -86,6 +86,8 @@ export const TEXT_INPUT_ELEMENT = 'Input.Text';
 export const TOGGLE_INPUT_ELEMENT = 'Input.Toggle';
 export const CHOICE_SET_INPUT_ELEMENT = 'Input.ChoiceSet';
 
+export const MAXIMUM_BYTES = 150000000; // 150 MB threshold limit
```

Comment:
Can you make this name more specific?
```suggestion
export const FILE_ATTACHMENT_MAX_SIZE = 150000000; // 150 MB threshold limit
```